sim/dex-data.ts

```diff
@@ -939,10 +944,12 @@ export class Move extends BasicEffect implements Readonly<BasicEffect & MoveData
 				}
 			}
 		}
-		if (this.category !== 'Status' && !this.zMovePower) {
+		if (this.category !== 'Status' && !this.zMovePower && !this.isZ) {
```

Comment:
Still TBD here, should we be giving isZ moves a zMovePower equal to 1, their base power, or something else? When is zMovePower used vs. the moves basePower?

Reply:
Apparently they can't be used at all, so their `zMovePower` should be 0, and this should be detectable by `canZMove`.

Reply:
```suggestion
		if (this.category !== 'Status' && !this.zMovePower) {
```

Reply:
Why should the zMovePower be 0 and not just absent?

Reply:
`0` means absent, here.

I suppose we could use `null` instead. I'm not opposed. Although note that the in-game representation is probably `0` since the games don't really have nullable ints.

Reply:
the type is `?`, undefined means absent....

Reply:
Like status moves are already `undefined` here